VirtualList/useThemeVirtualList.js

```diff
@@ -49,6 +49,12 @@ const useSpottable = (props, instances) => {
 
 	useSpotlightConfig(props, {spottable: mutableRef});
 
+	if (snapToCenter) {
```

Comment:
Can't we move the useEffect(..., [snapToCenter])?

Reply:
🆗